prow/plugins/releasenote/releasenote.go

```diff
@@ -212,11 +212,32 @@ func handlePullRequest(pc plugins.Agent, pr github.PullRequestEvent) error {
 	return handlePR(pc.GitHubClient, pc.Logger, &pr)
 }
 
-func handlePR(gc githubClient, log *logrus.Entry, pr *github.PullRequestEvent) error {
+func shouldIgnorePR(pr *github.PullRequestEvent) bool {
```

Comment:
Nit: The fact that a `true` return means the PR will be ignored instead of having a `shouldHandlePR` makes this hard to read for me at this time

Reply:
Method has been changed to shouldHandlePR